web-previewer/src/PixelAvatars.json

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
Suggestion: Perhaps we could move this to `src/contracts/PixelAvatarContract.json`